test.ts

```diff
@@ -228,8 +261,32 @@ describe('Component', () => {
         expect(spy).toHaveBeenCalledWith()
       })
     })
-
   })
+
+  function renderComponent(controller: Injectable<IControllerConstructor>) {
+    module('test', ['bcherny/ngimport'])
+      .component('myComponent', {
+        bindings: {
+          a: '<',
+          b: '<'
+        },
+        controller,
+        template: `{{a}}`
+      })
+
+    const el = element('<my-component a="a" b="b"></my-component>')
+    const parentScope = extend($rootScope.$new(true), {
```

Comment:
_.extend is required because the chrome version used in circle is apparently quite old